civiform/civiform
# Add "Edit program image" button to program details page.

### Description

This change adds a frontend skeleton for #5676. Specifically:
1) Adds an "Edit program image" button on the program edit homepage.
2) When that button is clicked, opens a new page under the route `admin/programs/{id}/image` to add/modify/remove the program image. This page currently just has a title and nothing else.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


#### New Features


### Instructions for manual testing

- Disable program_card_images flag then go to a program edit page -> verify "Edit program image" button doesn't appear
- Enable program_card_images flag then go to a program edit page -> verify "Edit program image" button appears.
- Click on "Edit program image" button -> verify a new page appears (with no content, yet :) )

The same steps should also work for setting up an eligibility/visibility condition and editing an existing eligibility/visibility condition.

### Issue(s)
Fixes #5996

Epic: #5676
